### DISCUSSION TITLE
Make hyperlinks more obvious

### DISCUSSION THREAD

**Discussion by EvilOlaf** [Lednerb/bilberry-hugo-theme#316] [Ideas]
Hey there,

I really enjoy this theme so far. Terrific job.

As an improvement I would like to make clickable hyperlinks more obvious because at the moment they are italic only and change color on mouse over.
I would like to change it in a way to have links always be shown in the hover color but I have to admit that I have no clue about CSS and such things. I assume the necessary changes need to be made in `_articles.scss` somewhere around line 379.
Some guidance in the correct direction would be appreciated.

Cheers
Werner

**Comment by igor-baiborodine** [comment]
Fixed with PR https://github.com/Lednerb/bilberry-hugo-theme/pull/335

**Reply by EvilOlaf** [reply]
Very nice
